konnectivity-client/go.mod

```diff
@@ -3,8 +3,8 @@ module sigs.k8s.io/apiserver-network-proxy/konnectivity-client
 go 1.17
 
 require (
-	github.com/golang/protobuf v1.4.3
-	google.golang.org/grpc v1.42.0
+	github.com/golang/protobuf v1.4.1
+	google.golang.org/grpc v1.27.0
```

Comment:
Can we add a comment explaining why we need this version and the same thing you pasted in the PR description with the grpc versions the different kube releases use?

Reply:
Good idea I'll add a comment.